### PR TITLE
feat/PRO-2401/trending-tokens-horizontal-tile

### PR DESCRIPTION
**Pull request by RanaBug**
## Description

- Frontend implementation of the Trending Tokens Tile

## How Has This Been Tested?

- New FE components for Trending Tokens tile (horizontal)
- FE tests of new components
- Components responsive to desktop, tablet and mobile

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/apps/pillarx-app/components/TrendingTokens/TrendingTokens.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    
+    const divRef = createRef()
+    const dimensions = useRefDimensions(divRef as React.RefObject<HTMLDivElement>)
+
+    const numberTrendingTokens = Math.floor(dimensions.width / trendingTokenWidth);
+
+    const DUMMY_TOKEN_VALUE = 160.12345678;
+
+    const DUMMY_TOKEN_PERCENTAGE = (tokenPriceChange: number) => 
+        (tokenPriceChange / DUMMY_TOKEN_VALUE) * 100;
```
Comment (IAmKio) [inline src/apps/pillarx-app/components/TrendingTokens/TrendingTokens.tsx:57]: Can i just confirm what these variables are being used for?

**Review by IAmKio — COMMENTED** [review]
LGTM - fantastic work, just one question then it should be good for approval 🙏

File: src/apps/pillarx-app/components/TrendingTokens/TrendingTokens.tsx (hunk 1 above)
Comment (RanaBug) [reply src/apps/pillarx-app/components/TrendingTokens/TrendingTokens.tsx:57]: The DUMMY_TOKEN_VALUE is being used for the token value in $ (until we add this in the api) and the DUMMY_TOKEN_PERCENTAGE is being calculated from the token value changes in the last 24h (which we have on the API, but then we are missing the token value) - This are placeholders essentially and will be replaced once the api is giving us the Token Value

**Review by IAmKio — APPROVED** [review]
LGTM! 🚀

**Pull request merged**